decidim/decidim
# Add attachments to accountability results

#### :tophat: What? Why?
Many municipalities/cities have requested to have attachments in the accountability results. Most other manageable resources can have attachments, so why not accountability results too?

I would see attachments especially important for the accountability results, even more so than for the budgeting projects. These can be e.g. project plans, concept drawings, illustrations, etc.

#### :pushpin: Related Issues
- Related to https://meta.decidim.org/processes/roadmap/f/122/proposals/15834

#### Testing
- Create a Decidim instance with the seed data
- Go to the admin panel to manage one participatory process with the accountability component
- Go to the accountability component
- See the attachment and folder buttons for the accountability results
- Add attachments and folders for a result
- Go to the participant view to see the attachments

#### :clipboard: Checklist

### :camera: Screenshots

For participants:
![accountability-attachments-01-participant](https://user-images.githubusercontent.com/864340/98836183-0e3d2480-244a-11eb-85dc-2a6097ca076c.jpg)

For admins:
![accountability-attachments-02-admin](https://user-images.githubusercontent.com/864340/98836145-054c5300-244a-11eb-9cc8-c918a04d79ba.jpg)
